content/guides/component-testing/mounting-angular.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Mounting Components
+---
+
+Many tests, regardless of framework or type, follow a similar format: **Arrange,
+Act, and Assert**. This pattern "Arrange, Act, Assert" was first coined in 2001
+by Bill Wilke and is explained thoroughly in his blog post
+["3A - Arrange, Act, Assert"](https://xp123.com/articles/3a-arrange-act-assert/).
+
+When it comes to component testing, mounting your component is where we
+**Arrange** our component under test. It is analogous to visiting a page in an
+end-to-end test.
+
+## What is the Mount Function?
+
+We ship a `mount` function for each front-end framework Cypress supports, which
+is imported from the `cypress` package. It is responsible for rendering
+components within Cypress's sandboxed iframe and handling and framework-specific
+cleanup.
+
+```ts
+import { mount } from 'cypress/angular'
+```
+
+### Using `cy.mount()` Anywhere
+
+While you can use the `mount` function in your tests, we recommend using
+[`cy.mount()`](/api/commands/mount), which is added as a
+[custom command](/api/cypress-api/custom-commands) in the
+**cypress/support/component.ts** file:
+
+<code-group>
+<code-block label="cypress/support/component.ts" active>
+
+```ts
+import { mount } from 'cypress/angular'
+
+Cypress.Commands.add('mount', mount)
+```
+
+</code-block>
+</code-group>
+
+This allows you to use `cy.mount()` in any component test without having to
+import the framework-specific mount command.
+
```

Comment:
This is missing the part about being able to customize the cy.mount command in the other guides. Not sure if it was removed on purpose or an error, but I think it makes sense here.